Humanizetime (#80)
trustpilot/ilenia: README.md

<TimeAgo date={date}/> // renders someting like "6 months ago"
```

### HumanizeTime

> Same output as TimeAgo without 'ago'. Transforms milli seconds to humainzed text.

Reviewer: ```suggestion
The component accepts a number milliseconds to human readable text. It renders the same as TimeAgo but without the 'ago' postfix. 
```

```javascript
import { HumanizeTime } from 'ilenia';

> <HumanizeTime time={700000}/>  // renders "10 minutes"

Reviewer: ```suggestion
<HumanizeTime time={600000}/>  // renders "10 minutes"
```

Reviewer: https://www.youtube.com/watch?v=X09oxyIeGuY

> const yesterday = new Date(new Date(date).setDate(date.getDate() + -1)) //

Reviewer: ```suggestion
const yesterday = new Date(new Date(date).setDate(date.getDate() + -1))
```
